src/form/formatters.js

```diff
@@ -10,6 +11,35 @@ export const prependSig = s => {
   return s.charAt(0) === '~' ? s : `~${s}`;
 };
 
+/**
+ * inserts a - every 7 characters
+ * (must be preceeded by prependSig to work correctly)
```

Comment:
That seems a bit icky. Maybe want an `ensurePatQFormat` or something? Probably not all that bad to leave this as is though.

Reply:
I was able to easily expand `ensurePatFormat` to cover the prependSig case as well